### PR TITLE
Add template for first test 01-BASIC

### PR DESCRIPTION
**Pull request by SergioAtSUSE**
- Related ticket: https://progress.opensuse.org/issues/45158
- Verification run: http://slindomansilla-vm.qa.suse.de/tests/1131

### CODE REVIEW

File: lib/main_common.pm [hunk 1]
```diff
@@ -2124,7 +2124,12 @@ sub load_security_tests {
 
 sub load_systemd_patches_tests {
     boot_hdd_image;
-    loadtest 'console/systemd_testsuite';
+    if (check_var('SYSTEMD_TESTSUITE', 'basic')) {
+        loadtest 'systemd_testsuite/test_01_basic';
```
Comment (rwx788) [inline lib/main_common.pm]: Are you going to have list of modules here? Alternative could be to pass perl module name in the variable and schedule it. So you won't need to change the code every time you want to change the schedule.

File: tests/systemd_testsuite/test_01_basic.pm [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+use base "consoletest";
+use warnings;
+use strict;
+use testapi;
+use utils 'zypper_call';
+use power_action_utils 'power_action';
+
+sub run {
+    select_console 'root-console';
```
Comment (rwx788) [inline tests/systemd_testsuite/test_01_basic.pm]: Are you just providing skeleton or you haven't pushed implementation which matches the description?

File: tests/systemd_testsuite/test_01_basic.pm [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+
```
Comment (jknphy) [inline tests/systemd_testsuite/test_01_basic.pm]: you can remove 1 line break here.

File: lib/main_common.pm (hunk 1 above)
Comment (SergioAtSUSE) [reply lib/main_common.pm]: I cannot assure now.
At the beginning I was thinking on creating a test suite per test, this is why the created has a specific name, it wouldn't be a list then, but a chain of "if else if".

I am trying to find a better solution by grouping all those test modules in one test suite, but for that I need to provide a clean environment for each one. "openQA snapshots" are not supported for s390x at the moment. (See [poo#46349](https://progress.opensuse.org/issues/46349))

If we can got it, my idea is to actually reuse the current SYSTEMD_TESTSUITE=1, but, until we finish the migration, I don't want to break it. So, at the moment this template is a mix of both. But, at least there is something that @tsaupe can clone and work on.

File: tests/systemd_testsuite/test_01_basic.pm (hunk 2 above)
Comment (SergioAtSUSE) [reply tests/systemd_testsuite/test_01_basic.pm]: Just a skeleton. @tsaupe will provide the implementation with my support.
The test suite will be schedule on the development job group

**Comment by SergioAtSUSE** [comment]
Migratio follows on: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6593

**Pull request merged**